ideascube/serveradmin/tests/test_views.py

```diff
@@ -642,3 +642,29 @@ def test_staff_forgets_non_existing_connection(mocker, staffapp):
     assert u'No known Wi-Fi networks' not in res.unicode_body
     assert u'my home network' in res.unicode_body
     assert u'random open network' in res.unicode_body
+
+
+@pytest.mark.usefixtures('installed_packages_catalog')
+def test_staff_should_see_packages_with_cards(staffapp, systemuser):
+    from ideascube.configuration import set_config
+    set_config('ideascube', 'displayed-package-ids', ['test.package1'], systemuser)
+    res = staffapp.get(reverse("server:cards"), status=200)
+    assert 'Test package1' in res.unicode_body
+    assert 'Test package2' in res.unicode_body
+    assert 'Test package3' not in res.unicode_body
+
+    form = res.forms['cards']
+    inputs = form.fields['visible']
+    for _input in inputs:
+        should_be_checked = (_input._value == 'test.package1')
```

Comment:
> Why don't you give an id to each input? That would allow you to identify them uniquely.

Cause this id would be used only for testing. And I would like not modify the generated html just to allow a test to be clever written.

> I mean, it's the same in the view, you don't get in POST a list of all the checkbox with a checked attribute set to True or False, you only get the list of values of the checked checkboxes.

I'm not testing the POST request. I'm testing the generated html.
And all the logic is base on the value attribute. If I start testing id attribute, I'm not testing the right thing.